README.rst

```diff
@@ -1,22 +1,24 @@
-Webhooks for `Open edX`_, integrating `JIRA`_ and `Github`_,
-designed to be deployed onto `Heroku`_.
+Open edX Webhooks
+=================
 
-|build-status| |coverage-status| |docs|
+Webhooks for `Open edX <http://openedx.org>`__ integrating
+`JIRA <https://openedx.atlassian.net>`__ and
+`Github <https://github.com/edx>`__. Designed to be deployed onto
+`Heroku <http://heroku.com>`__.
```

Comment:
Since I plan on updating documentation for this repo, I decided on a workflow that works for me, which is to write in markdown then convert to reStructuredText. This is an artifact of the conversion, which I can correct manually if you feel strongly about it.